CairoMakie/src/primitives.jl

```diff
@@ -587,6 +565,11 @@ function draw_glyph_collection(
         # offsets and scale apply in markerspace
         gp3 = glyph_pos[Vec(1, 2, 3)] ./ glyph_pos[4] .+ model33 * (glyphoffset .+ p3_offset)
 
+        if any(isnan, gp3)
```

Comment:
This is the fix for https://github.com/MakieOrg/Makie.jl/pull/3210#issuecomment-1778933826.